### ISSUE TITLE
Instructions for how to install apps into a filesystem with the emulator

### ISSUE THREAD

**Issue by jwise** [pebble-dev/RebbleOS#75]
It would be nice to have some documentation for how to install apps into the filesystem with the emulator, and then how to take that emulator filesystem and test RebbleOS with it, to get users off the ground faster with finding bugs and missing APIs, and fixing them!

**Comment by eevui** [comment]
Blocked by #71 ?

**Comment by jwise** [comment]
Eventually, yes :-) But for right now we install real apps using PebbleOS, then copy the post-installation filesystem image from the correct directory over into the base image that we use for RebbleOS.
